scripts/qa_crowbarsetup.sh

```diff
@@ -4116,8 +4116,17 @@ function oncontroller_testsetup
     fi
 
     # this file is created by the designate barclamp
-    designate_pools="/etc/desigate/pools.crowbar.yaml"
-    [[ -e $designate_pools ]] && designate-manage pool update --file $designate_pools
+    designate_pools="/etc/designate/pools.crowbar.yaml"
+    if [[ -e $designate_pools ]]; then
+        designate-manage pool update --file $designate_pools
+        # default values from the node.neutron.dns_domain
+        openstack tld create --name local
+        openstack zone create --email admin@example.org openstack.local.
+        theport=$(openstack port create --network fixed --dns-name iamgroot thanos -f value -c id)
+        openstack floating ip create floating --port $theport
+        wait_for 30 5 "nslookup iamgroot.openstack.local >> /dev/null" "thanos port is nslookup-able"
+        [ $want_tempest = 1 ] && openstack tld create --name com # needed by designate-template-plugin smoke test
```

Comment:
```suggestion
        [[ $want_tempest = 1 ]] && openstack tld create --name com # needed by designate-template-plugin smoke test
```

to be consistent with the rest..